python/rascaline/profiling.py

```diff
@@ -19,6 +19,7 @@ class Profiler:
 
         with rascaline.Profiler() as profiler:
             # run some calculations
+            pass
```

Comment:
```suggestion
            ...
```

I think this is valid Python, and looks a bit better for examples!

Reply:
Yes this is better!